automation/test.sh

```diff
@@ -36,12 +36,16 @@ if [[ $TARGET =~ openshift-.* ]]; then
   else
     export KUBEVIRT_PROVIDER="os-3.10.0"
   fi
-elif [[ $TARGET =~ .*-1.9.3-.* ]]; then
-  export KUBEVIRT_PROVIDER="k8s-1.9.3"
+elif [[ $TARGET =~ .*-1.10.4-.* ]]; then
+  export KUBEVIRT_PROVIDER="k8s-1.10.4"
 else
-  export KUBEVIRT_PROVIDER="k8s-1.10.3"
+  export KUBEVIRT_PROVIDER="k8s-1.11.0"
 fi
 
+# network settings for kubeadm preflight check (ignore failures on old CI)
```

Comment:
Something like this needs to be part of the image itself. What you do here is not even executed inside of the cluster  images. Also since we have full control over the qemu-vms, it should never benecessary to add something like `|| true` to account for different qemu-vms.